src/SAWScript/Prover/MRSolver/Monad.hs

```diff
@@ -790,9 +799,9 @@ mrGetCoIndHyp :: FunName -> FunName -> MRM (Maybe CoIndHyp)
 mrGetCoIndHyp nm1 nm2 = Map.lookup (nm1, nm2) <$> mrCoIndHyps
 
 -- | Run a compuation under an additional co-inductive assumption
-withCoIndHypRaw :: CoIndHyp -> MRM a -> MRM a
-withCoIndHypRaw hyp m =
-  do debugPretty 1 ("withCoIndHyp" <+> ppInEmptyCtx hyp)
+withCoIndHyp :: CoIndHyp -> MRM a -> MRM a
+withCoIndHyp hyp m =
+  do debugPretty 2 ("withCoIndHypRaw" <+> ppInEmptyCtx hyp)
```

Comment:
This should probably be kept as `"withCoIndHyp"` given that it's now the name of this function.

Reply:
Whoops, yes, it should be